src/pages/about/index.md

```diff
@@ -4,35 +4,35 @@ title: About Us
-heading: About
-description: >-
+  to share their time and talents to further the Kingdom of God.
+headingAboutTechConnect: About Tech Connect
+aboutTechConnect: >-
   Tech Connect is a webservice meant for connecting developers and the minds
-  behind projects, run by Mission Aviation Fellowship (MAF). MAF is all about
-  making connections -- and technology plays a key roles. We want to introduce
-  isolated people to Jesus Christ through a network of technology and
-  discipleship strategies. Technology constantly changes, so MAF Tech Resources
+  behind projects, run by Mission Aviation Fellowship (MAF). MAF wants to
+  ensure that no technically skilled person feels their skills have no impact
+  and that no mission minded person feels unequipped to accomplish their goals
+  of furthering the Kingdom of God. MAF believes that both of these goals can
+  be met through Tech Connect!
+image2:
+  alt: Coffee beans
+  image: /img/AboutTechConnect.jpg
+headingAboutMAF: About MAF
+aboutMAF: >-
+  MAF is all about making connections -- and technology plays a key roles. We 
+  want to introduce isolated people to Jesus Christ through a network of technology 
+  anddiscipleship strategies. Technology constantly changes, so MAF Tech Resources
   is always adapting to create opportunities to connect people to Jesus. We use
   technology in a variety of ways in order to bring God's Word into the most
   remote parts of the world; providing connectivity through email and VPN
   services, and VSAT and HF/VHF radio installations; digital solutions for
   missions like mobile apps with the Bible and Christian materials; and business
-  services that support our flight programs around the world. Tech Connect is a
-  webservice meant for connecting developers and the minds behind projects, run
-  by Mission Aviation Fellowship (MAF). MAF is all about making connections --
-  and technology plays a key roles. We want to introduce isolated people to
-  Jesus Christ through a network of technology and discipleship strategies.
-  Technology constantly changes, so MAF Tech Resources is always adapting to
-  create opportunities to connect people to Jesus. We use technology in a
-  variety of ways in order to bring God's Word into the most remote parts of the
-  world; providing connectivity through email and VPN services, and VSAT and
-  HF/VHF radio installations; digital solutions for missions like mobile apps
-  with the Bible and Christian materials; and business services that support our
-  flight programs around the world.
+  services that support our flight programs around the world.
 quote: >-
-  It's the best way to connect with people who can truly help you. There's a
+  "It's the best way to connect with people who can truly help you. There's a
   class of people called 'early adopters', but Tech Connect sponsors are so much
-  earlier than that!
-image2: /img/chemex.jpg
+  earlier than that!"
+image3:
+  alt: Coffee beans
```

Comment:
"Coffee beans"? Can we make this "MAF logo" or something equally informative?

Reply:
Agh, yes, we definitely can! My bad on overlooking that one...